src/modules/commander/commander.cpp

```diff
@@ -2698,19 +2698,20 @@ int commander_thread_main(int argc, char *argv[])
 			     internal_state.main_state == commander_state_s::MAIN_STATE_STAB ||
 			     internal_state.main_state == commander_state_s::MAIN_STATE_ALTCTL ||
 			     internal_state.main_state == commander_state_s::MAIN_STATE_POSCTL) &&
-			    ((status.rc_signal_lost && status_flags.gps_failure) ||
+			    ((status.rc_signal_lost /*&& status_flags.gps_failure*/) ||
```

Comment:
I don't think we want termination if we lose RC and still have GPS. The question is probably what the meaning of `gps_failure` is or should be.

It could mean the following:
1. GPS not available, or GPS available once and now lost.
or:
2. GPS available once and now lost.